cli/smartcontract/testdata/rpcbindings/notifications/config_extended.yml

```diff
@@ -26,6 +26,11 @@ events:
         extendedtype:
           base: Struct
           name: crazyStruct
+          fields:
+            - field: I
+              base: Integer
+            - field: B
+              base: Boolean
```

Comment:
Why do you include `fields` in the definition of named extended type? It shouldn't be like that, all named types should be specified in the `namedtypes` section, and extended type here should just point to a specific named type.